nixos/tests/nginx-proxyprotocol/generate-certs.nix

```diff
@@ -0,0 +1,34 @@
+# Minica can provide a CA key and cert, plus a key
+# and cert for our fake CA server's Web Front End (WFE).
+{
+  pkgs ? import <nixpkgs> {},
+  minica ? pkgs.minica,
+  mkDerivation ? pkgs.stdenv.mkDerivation
+}:
+let
+  conf = import ./snakeoil-certs.nix;
+  domain = conf.domain;
+  domainSanitized = pkgs.lib.replaceStrings ["*"] ["_"] domain;
+in mkDerivation {
+  name = "test-certs";
+  buildInputs = [ (minica.overrideAttrs (old: {
+    prePatch = ''
```

Comment:
```suggestion
    postPatch = ''
```

Reply:
Are you certain of this suggestion? This is what has been written in the original ACME files.

Reply:
I applied it, hopefully, it should not break but it creates divergence from ACME original code.